cmake/macros/PlatformIntrospection.cmake

```diff
@@ -0,0 +1,122 @@
+# Returns a lowercased version of a given lsb_release field.
```

Comment:
Is this code taken from an example somewhere?

Reply:
Yep, taken from here https://github.com/couchbase/tlm
As I've checked there is no licensing. Can we use it?

Reply:
ok